packages/web/src/components/track/TrackBannerIcon.tsx

```diff
@@ -7,36 +8,52 @@ import styles from './TrackBannerIcon.module.css'
 
 export enum TrackBannerIconType {
   STAR = 'star',
-  HIDDEN = 'hidden'
+  HIDDEN = 'hidden',
+  LOCKED = 'locked',
+  COLLECTIBLE_GATED = 'collectible gated',
+  SPECIAL_ACCESS = 'special access'
 }
 
 const TrackBannerIcon = ({
   type,
   isMobile,
-  isMatrixMode
+  isMatrixMode,
```

Comment:
Feels like we shouldn't have to do this.. is this something that happens elsewhere as well? Ok to merge this for now but should maybe investigate further.

Reply:
hmm not sure why you think we shouldn't have to do this. this component is used by different other components, and we sometimes need to override